### PR TITLE
fix: reward cg prices

### PR DESCRIPTION
**Pull request by fredwes**
Previously only esSEAM and SEAM were fetching reward USD price from CG. Now make all reward tokens that don't have oracle automatically fetch from CG

### CODE REVIEW

**Review by WingsDevelopment — DISMISSED** [review]

File: src/store/incentiveSlice.ts [hunk 1]
```diff
@@ -18,82 +20,84 @@ export interface IncentiveSlice {
...
+      vIncentiveData: await incentiveDataInjectRewardPriceUSD(reserveIncentive.vIncentiveData),
+      sIncentiveData: await incentiveDataInjectRewardPriceUSD(reserveIncentive.sIncentiveData),
+    }))
+  );
 
-const userIncentiveDataInjectSEAMPriceUSD = (
-  incentiveData: UserIncentiveDataHumanized,
-  seamPriceUSD: string
-): UserIncentiveDataHumanized => ({
+  console.log("reps", reps);
```
Comment (kitanovicd) [inline src/store/incentiveSlice.ts]: console.log?

File: src/store/utils/externalPriceFetcher.ts [hunk 2]
```diff
@@ -46,13 +56,21 @@ const getCoinGeckoPriceUSD = async (address: string): Promise<string> => {
 
 const getCoinGeckoCoinDetails = async (address: string): Promise<string | undefined> => {
   try {
-    const resp = await fetch(`https://cg-price-cache.seamlessprotocol.com/coins/id/contract/${address}`);
+    address = REPLACE_ADDRESSES[address.toLowerCase()]
```
Comment (kitanovicd) [inline src/store/utils/externalPriceFetcher.ts]: Can this be written something like REPLACE_ADDRESS[address.toLowerCase()] || address

File: src/store/utils/externalPriceFetcher.ts [hunk 3]
```diff
@@ -13,6 +13,10 @@ const queryClient = new QueryClient({
 
 export const CHAIN_ID = 'base';
 export const PRICE_FEED_DECIMALS = 8;
+const IGNORE_ADDRESSES = ['0x5607718c64334eb5174CB2226af891a6ED82c7C6'];
+const REPLACE_ADDRESSES: { [key: string]: string } = {
+  ['0x998e44232bef4f8b033e5a5175bdc97f2b10d5e5']: '0x1C7a460413dD4e964f96D8dFC56E7223cE88CD85',
+};
 
 export const fetchExternalPrice = async (
   address: string
```
Comment (kitanovicd) [inline src/store/utils/externalPriceFetcher.ts:22]: Can we do .toLowerCase() in the beginning of the function so we don't need to do .lowerCase() every time we call it?

File: src/store/incentiveSlice.ts (hunk 1 above)
Comment (fredwes) [reply src/store/incentiveSlice.ts]: Ah yes forgot to remove that. Thanks. REmoved

File: src/store/utils/externalPriceFetcher.ts (hunk 2 above)
Comment (fredwes) [reply src/store/utils/externalPriceFetcher.ts]: Done.

File: src/store/utils/externalPriceFetcher.ts (hunk 3 above)
Comment (fredwes) [reply src/store/utils/externalPriceFetcher.ts:22]: Done

**Review by kitanovicd — APPROVED** [review]

**Pull request merged**